core/test/Test/Pos/Core/ExampleHelpers.hs

```diff
@@ -118,57 +118,50 @@ import qualified Serokell.Util.Base16 as B16
 
 import qualified Cardano.Crypto.Wallet as CC
 import           Pos.Binary.Class (Raw (..), asBinary)
-import           Pos.Data.Attributes (Attributes, mkAttributes)
 import           Pos.Core.Common (AddrAttributes (..), AddrSpendingData (..),
-                     AddrStakeDistribution (..), Address (..), BlockCount (..),
-                     ChainDifficulty (..), Coeff (..), Coin (..),
-                     CoinPortion (..), IsBootstrapEraAddr (..), Script (..),
-                     ScriptVersion, SharedSeed (..), SlotLeaders,
-                     StakeholderId, StakesList, TxFeePolicy (..),
-                     TxSizeLinear (..), addressHash, coinPortionDenominator,
-                     makeAddress, makePubKeyAddress, mkMultiKeyDistr)
+                                  AddrStakeDistribution (..), Address (..), BlockCount (..),
+                                  ChainDifficulty (..), Coeff (..), Coin (..), CoinPortion (..),
+                                  IsBootstrapEraAddr (..), Script (..), ScriptVersion,
+                                  SharedSeed (..), SlotLeaders, StakeholderId, StakesList,
+                                  TxFeePolicy (..), TxSizeLinear (..), addressHash,
+                                  coinPortionDenominator, makeAddress, makePubKeyAddress,
+                                  mkMultiKeyDistr)
```

Comment:
This stylish-haskell config is wrong.

Reply:
This PR is against the `release/1.3.1` branch, so the stylish-haskell rules are the old ones.

Reply:
^ what Erik said. It bothered me too, but I didn't want to reformat the whole repo 🙃